internal/distribution/supported_versions.go

```diff
@@ -17,14 +18,31 @@ import (
 	"github.com/sighupio/furyctl/internal/git"
 )
 
+const (
+	RequiredSegments          = 2
```

Comment:
I wouldn't export these, as this is an internal constant.
```suggestion
	requiredSegments          = 2
```